geocity/apps/submissions/templatetags/submissions_extras.py

```diff
@@ -69,7 +69,7 @@ def submission_summary(context, submission):
     geo_time_formset = SubmissionGeoTimeFormSet(
         None,
         form_kwargs={"submission": submission, "disable_fields": True},
-        queryset=submission.geo_time.all(),
+        queryset=submission.geo_time.filter(form=None, field=None).all(),
```

Comment:
Could be impacted by model discussion.
`.filter(submission__forms__isnull=True, submission__forms__fields__isnull=True)` (This is just the logic, could be written differently)

Reply:
Won't work , better: `.filter(forms__isnull=True, forms__fields__isnull=True)`